playgrounds/skeleton-react/src/app/components/tree-view/page.tsx

```diff
@@ -0,0 +1,96 @@
+'use client';
+
+import { TreeView, createTreeViewCollection } from '@skeletonlabs/skeleton-react';
+import { ChevronDownIcon, FileIcon, FolderIcon } from 'lucide-react';
+
+interface Node {
+	id: string;
+	name: string;
+	children?: Node[];
+}
+
+function TreeNode(props: { node: Node; indexPath: number[] }) {
+	return (
+		<TreeView.NodeProvider value={props}>
+			{props.node.children ? (
+				<TreeView.Branch>
+					<TreeView.BranchControl>
+						<TreeView.BranchIndicator>
+							<ChevronDownIcon className="size-4" />
+						</TreeView.BranchIndicator>
+						<TreeView.BranchText>
+							<FolderIcon className="size-4" />
+							{props.node.name}
+						</TreeView.BranchText>
+					</TreeView.BranchControl>
+					<TreeView.BranchContent>
+						<TreeView.BranchIndentGuide />
+						{props.node.children.map((childNode, childIndex) => (
+							<TreeNode key={childNode.id} node={childNode} indexPath={[...props.indexPath, childIndex]} />
+						))}
+					</TreeView.BranchContent>
+				</TreeView.Branch>
+			) : (
+				<TreeView.Item>
+					<FileIcon className="size-4" />
+					{props.node.name}
+				</TreeView.Item>
+			)}
+		</TreeView.NodeProvider>
+	);
+}
+
+export default function Page() {
+	const collection = createTreeViewCollection<Node>({
+		nodeToValue: (node) => node.id,
+		nodeToString: (node) => node.name,
+		rootNode: {
```

Comment:
We're probably doing more than enough to promote Zag and their related products. Let's make the placeholder data a bit more generic. I'd suggest mimicking a nested navigation structure, something like directories and images on a CDN or something along those lines.